app/views/course/assessment/answer/_guided.html.slim

```diff
@@ -0,0 +1,2 @@
+- if answer.attempting?
+  = base_answer_form.button :button, t('common.submit'), value: answer.id, name: 'submit_answer'
```

Comment:
Actually I just checked. jQuery 2 does not support IE8, only IE9. So we have other problems.

If that's the case, using button is fine. But then call the button `submitted_answer_id` or something.